visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -66,6 +66,11 @@ export enum NodeType {
 	FOLDER = "Folder"
 }
 
+export enum SortingOption {
+	Name = "Name",
+	Childnodes = "Childnodes"
```

Comment:
I would prefer another string instead of "Number of Files", as the string is rather long, making a distinction between the sorting dialog and the matching-file-counter rather difficult
I will try to come up with one, otherwise proposals are really welcome

Reply:
but isn't the matching-file-counter and the sorting dialog based on the same metric? 
I don't really see a reason to make a distinction here. Can you give me an example?

Reply:
I mean in regards to the ui, the button get pretty close to the matching-file-counter icons
My bad on the wording here